extension/textext/asktext.py

```diff
@@ -899,20 +907,20 @@ def create_buttons(self):
             button_box.set_spacing(spacing)
 
             self._cancel_button = Gtk.Button(stock=Gtk.STOCK_CANCEL)
-            self._cancel_button.set_tooltip_text("Don't save changes")
+            self._cancel_button.set_tooltip_text("Don't save changes (ESC)")
```

Comment:
Shouldn't it be formatted same way as above?

Reply:
What do you mean?

Reply:
ESC is not the only possible option. It might be `Ctrl+Q`, etc

Reply:
This is handled here (called when the settings are changed or loaded from config.json):
https://github.com/textext/textext/blob/705e47c39896cdf7a150cbe834daa8fb24b3a1b8/extension/textext/asktext.py#L701-L705

Reply:
Then this line is useless and should be deleted.

Reply:
Unfortunately not. During dialog creation ``close_shortcut_cb`` is not called when we have the default behavior (ESC).

It is not clean due to historic development of the code. I know.

I will address this in a separate PR,